vectorlm/utils/save_utils.py

```diff
@@ -275,4 +275,4 @@ def load_scheduler(
     print(f"Loading scheduler state from {input_scheduler_file}")
     state_dict = torch.load(input_scheduler_file)
     scheduler.load_state_dict(state_dict)
-    print(f"Scheduler state loaded from {input_scheduler_file}")
+    print(f"Scheduler state loaded from {input_scheduler_file}")
```

Comment:
I believe new lines at the end of files are required by ruff

Reply:
Applied ruff formatting as requested.